aarch64-options.mk

```diff
@@ -0,0 +1,13 @@
+#
+# Copyright (c) 2018 ARM Limited
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# The time limit in seconds for each test
+TIMEOUT := 120
+
+# union for 'make test'
+UNION := $(shell bash -f .ci/filter_test_aarch64.sh)
+
+# selected test suites for docker integration tests
+FOCUS := $(shell bash -f .ci/filter_docker_aarch64.sh)
```

Comment:
I guess you will file some issues to track the tests are not passing on ARM, right?

Reply:
@devimc We want to create one issue to track the tests are not passing on Arm. In this issue, we will list all nonworking tests, and link PRs to fix them. Create each issue for tests will keep too many flying issues, How do you think about it?

Reply:
I agree